src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -0,0 +1,94 @@
+
+- Search condition 2: `!event.type:error release.version:{another_version}`
+
+You can also compare performance before/and after a certain version with the following conditions:
+
+- Search condition 1: `!event.type:error release.version:<{version}`
+- Search condition 2: `!event.type:error release.version:>={version}`
+
+
+### Response thresholds
+
+The "Distribution Duration" widget shows the spread of transaction duration times. This visualization can be modified to show if transaction durations are within certain target thresholds and present them as "satisfactory", "tolerable", or "frustrating" transactions. To create this type of visualization, change the "Visualization Display" to "Area Chart". Area charts stack results and are more appropriate for results that are cumulative:
+
+- Visualization Display: `Area Chart`
+
+Set one of the y-axis values to `count()` and remove the other two axes:
+
+- Y-Axis: `count()`
+
+To the first query, add the search condition for satisfactory transaction duration (this example uses 300ms as the satisfactory response threshold):
+
+- Search condition 1: `!event.type:error transaction.duration:<300`
+
+Add another query for the tolerable condition by clicking the "Add Query" button. In this example, we're using the [Apdex](/product/performance/metrics/#apdex) definition where tolerable response times are between the satisfactory threshold and four times the satisfactory threshold:
+
+- Search condition 2: `!event.type:error (transaction.duration:<1200 AND transaction.duration:>=300)`
+
+Finally, add a third query for for the frustrating transaction duration:
+
+- Search condition 3: `!event.type:error transaction.duration:>=1200`
+
+The chart now shows cumulative counts at different response time thresholds.
+
+![Response Threshold Widget.](dashboard-response-threshold-widget.png)
+
+### Top issues
+
+You can track the impact of your issues by counting the error events associated with each unique issue. Modify the "Latest Unresolved Issues" widget by selecting the "All Events" data set. Update the search condition to filter for error events:
+
+- Data Set: `All Events`
+- Search condition: `event.type:error`
+
+Update columns to group by `issues` and apply `count()` as the aggregating function. Order count in descending order so the widget displays issues with the most error events:
+
+- Columns: `issue, count(), any(message)`
+- Sort by: `count() desc`
+
+### Unique issues by projects
+
+Monitoring the cardinality of issues alongside the total number of errors in your projects can give you a sense of the impact of issues in your project. For example, low cardinality in issues and a high number of total errors indicates you have a few very high impact issues. Modify the "Latest Unresolved Issues" widget by selecting the "All Events" data set. Update the search condition to filter to error events:
```

Comment:
I think just defining cardinality here so we can keep on using it makes the most sense.
```suggestion
Monitoring the cardinality of issues (the number of unique errors in an issue) alongside the total number of errors in your projects can give you a sense of the impact of issues in your project. For example, low cardinality  in issues and a high number of total errors indicates you have a few very high impact issues. Modify the "Latest Unresolved Issues" widget by selecting the "All Events" data set. Update the search condition to filter to error events:
```

Reply:
Updated here https://github.com/getsentry/sentry-docs/pull/4595/commits/14fb76a203a5e4c49c0be6affd312009de562d9f
(definition slightly different)